README.md

```diff
@@ -40,10 +40,11 @@ Enjoy your new RPM.
 
 Building a .deb should be straight forward with (Tested on Ubuntu 15.04):
 ```
+# apt-get install libnm-glib-dev libnm-glib-vpn-dev libnm-util-dev libgnome-keyring-dev libdbus-glib-1-dev dh-autoreconf libgtk-3-dev sshpass
```

Comment:
Ideally that'd come from https://github.com/danfruehauf/NetworkManager-ssh/blob/master/debian/control - this specified all the dependencies one might need. But that's better than nothing. Unless you think we might want to add a comment to look at `debian/control`?

As a side note I'm saying that all the time, having the debian/ubuntu build instructions here is a bit of an abomination, best is to have a package for that maintained upstream.